chain-impl-mockchain/tests/common/arbitrary/output.rs

```diff
@@ -0,0 +1,28 @@
+use super::KindWithoutMultisig;
+use chain_addr::{Address, Discrimination};
+use chain_impl_mockchain::transaction::Output;
+use quickcheck::{Arbitrary, Gen};
+
+#[derive(Clone, Debug)]
+pub struct OutputsWithoutMultisig(pub Vec<Output<Address>>);
```

Comment:
Again, there could be a filter fn to get this without added types, in iterator style:

```rust
fn kind_without_multisig(item: Kind) -> Option<Kind> {
    use Kind::*; 
    match input {
        Multisig => None,
        x => Some(x),
    }
}
```

```rust
iter::from_fn(|| Some(Kind::arbitrary(gen)))
    .filter_map(kind_without_multisig)
    .take(n)
```

Reply:
The generator might be difficult to use in this style, if needed in multiple closures.